src/views/MainView.vue

```diff
@@ -224,6 +209,25 @@ export default {
     },
   },
   methods: {
+    async updateAuth() {
```

Comment:
I see this was copied from the default layout, but does it ever get called?

Reply:
@dacoaster Not sure if I'm reading this correctly but I think this was actually added in the hotkeys work:

commit a58c3db88f04bdcaebdf6bb509ebe57a4be7a34e
Author: dacoaster <david@yatt.ai>
Date:   Mon Aug 28 06:17:56 2023 -0700

    Finishing hotkeys and removing minimize

Does that seem correct? 

git diff a58c3db88f04bdcaebdf6bb509ebe57a4be7a34e src/views/MainView.vue

![image](https://github.com/yatt-ai/yattie/assets/28798717/725f3b89-e506-4859-a403-348175b6f466)

Reply:
@dacoaster Bump on this one. Does this belong here from the hotkeys work?